include/pm/state.h

```diff
@@ -48,10 +48,10 @@ enum pm_state {
 	 * @brief Suspend to idle state
 	 *
 	 * The system goes through a normal platform suspend where it puts
-	 * all of the cores in deepest possible idle state and *may* puts peripherals
```

Comment:
> stick to 80 cols

per guidelines, max is 100 characters per line. So why are we changing this?

Reply:
file was not using 100char width before (most lines wrap at 80cols except 2 or 3), so I think making it all 80 cols improves consistency